packages/rosaenlg/test/test-rosaenlg/fr_FR/value.pug

```diff
@@ -151,3 +160,12 @@ t
 
   //- debug
   l #[+value('alsacien', {adj:'vieux', adjPos:'BEFORE', number:'P' } )]
+
+  //- adjectifs possessifs

```

Comment:
a for loop or a eachz could make the code a little more compact

Reply:
done